fapolicy_analyzer/ui/subject_list.py

```diff
@@ -106,7 +105,22 @@ def _columns(self):
         fileColumn.set_sort_column_id(2)
         return [trustColumn, accessColumn, fileColumn]
 
+    def _trust_markup(self, subject):
+        u_str = "U"
+        at_str = '<span color=\"red\"><b>AT</b></span>'
+        st_str = '<span color=\"red\"><b>ST</b></span>'
+
+        if subject.trust.lower() == "at":
+            at_str = '<span color=\"green\"><b><u>AT</u></b></span>'
+        elif subject.trust.lower() == "st":
+            st_str = '<span color=\"green\"><b><u>ST</u></b></span>'
+        elif subject.trust.lower() == "u":
+            u_str = '<span color=\"green\"><b><u>U</u></b></span>'
```

Comment:
Its not enough here to just look at the `subject.trust`. You need to also look at the `subject.trust_status`. The `trust` only tells which trust database the file appears in. The `trust_status` tells if it is trusted in that database or not.